src/uct/cuda/base/cuda_iface.c

```diff
@@ -18,9 +18,14 @@ uct_cuda_base_query_devices_common(
 {
     ucs_sys_device_t sys_device = UCS_SYS_DEVICE_ID_UNKNOWN;
     CUdevice cuda_device;
+    ucs_status_t status;
 
     if (cuCtxGetDevice(&cuda_device) == CUDA_SUCCESS) {
-        uct_cuda_base_get_sys_dev(cuda_device, &sys_device);
+        status = uct_cuda_base_get_sys_dev(cuda_device, &sys_device);
+        /* Need this check to pass coverity */
+        if (status != UCS_OK) {
+            sys_device = UCS_SYS_DEVICE_ID_UNKNOWN;
```

Comment:
maybe this should be done inside uct_cuda_base_get_sys_dev? so it will return void